Developers/Writing_Bears.rst

```diff
@@ -65,14 +65,15 @@ This bear is stored at ``./bears/HelloWorldBear.py``
 In order to let coala execute this bear you need to let coala know where
 to find it. We can do that with the ``-d`` (``--bear-dirs``) argument:
 
-``coala -f src/*.c -d bears -b HelloWorldBear -L DEBUG``
+``coala -f src/*.c -d bears -b HelloWorldBear -L DEBUG --flush-cache``
 
 .. note::
 
     The given bear directories must not have any glob expressions in them. Any
     character that could be interpreted as a part of a glob expression will be
     escaped. Please use comma separated values to give several such
-    directories instead.
+    directories instead. Do not forget to flush the cache if you set a new
+    bear on a file which has previously been already analyzed.
```

Comment:
also give info, such as: "Do not forget to flush the cache (by adding the argument ``--flush-cache``) if you .."